docs/authentication/saml-at-clerk.mdx

```diff
@@ -134,7 +134,7 @@ Everything should be set up now for you to try out authentication via SAML. Go t
 
 **I’ve enabled other strategies but they don’t work.**
 
-It is expected that once Enterprise SSO is enabled in an application, there should be no other authentication methods applicable. This is in line with an organization’s intent to manage their users’ identity from one place.
+A Clerk application can have multiple authentication strategies, but an organization that enables Enterprise SSO can not. Once Enterprise SSO is enabled for an organization, there can be no other authentication methods for that organization. This is in line with an organization’s intent to manage their users’ identity from one place. This will allow your Clerk application to enable Enterprise SSO connections for certain organizations while others use non-Enterprise SSO methods depending on each organization's needs.
```

Comment:
@royanger When SAML is enabled, it is bound to an email domain (e.g. clerk.com). Authentication strategies for that domain are short-circuited in favor of SAML. All other email domains can use the rest of the authentication strategies.

So the phrasing should be around the email domain, not the organization yet.